dockstore-language-plugin-parent/src/main/java/io/dockstore/language/MinimalLanguageInterface.java

```diff
@@ -69,7 +69,8 @@
     /**
      * When indexing, Dockstore will distinguish between extra files that hold things like extra code, tools, configuration (imported descriptors) and test parameter files (example parameter sets used to run a workflow)
      */
-    enum GenericFileType { IMPORTED_DESCRIPTOR, TEST_PARAMETER_FILE, CONTAINERFILE }
+    enum GenericFileType { IMPORTED_DESCRIPTOR, TEST_PARAMETER_FILE, CONTAINERFILE
+    }
```

Comment:
Interesting how they format that. I don't think I like it, but oh well.

Reply:
I think it's supposed to be:
```
enum GenericFileType {
        IMPORTED_DESCRIPTOR, TEST_PARAMETER_FILE, CONTAINERFILE
    }
```
but checkstyle and codestyle doesn't seem to enforce that or autofix it in that way